### ISSUE TITLE
Define Project mission

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#70]
## User story

1. As a PO
2. I want to define a Project mission
3. So that the team has a better understanding of how our project should run

## Acceptance criteria

- [ ] A Project mission is defined 
- [ ] The Project mission is filled in the planning document 
- [ ] The Project mission is presented to the IP and accepted by IP

## Definition of done (DoD)

- [ ] Added only after week 5
- [ ] The same for all features
- [ ] Here goes the project specific part

## DoD general criteria

- [ ] Feature has been fully implemented
- [ ] Feature has been merged into the mainline
- [ ] All acceptance criteria were met
- [ ] Product owner approved features
- [ ] All tests are passing
- [ ] Developers agreed to release

## Additional details

- Task is related to irregular work.
